rest-api-security/src/main/resources/schema.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE persons(
+                          person_id          bigint primary key,
+                          person_name        varchar(128)  not null,
+                          person_surname     varchar(128) not null,
+                          person_birthday    date not null
+);
+
+CREATE TABLE subscriptions
+(
+    subscription_id    bigint primary key,
+    person_id  bigint not null,
```

Comment:
Optional: create an index for that field

Reply:
Done